### DISCUSSION TITLE
AutoComplete on Scroll load Data pagination vise  - API Base Send Request

### DISCUSSION THREAD

**Discussion by tayyabroy786** [primefaces/community#1337] [PrimeNG] [answered]
Hi,
I was working on this but not found any functionality about scrolling to handle on scroll send request to fetch data for next record.
Can we implement this feature thanks.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can use virtualScroll + lazy

**Comment by tayyabroy786** [comment]
how it will hit next next page on scroll ?
